key.core/src/main/resources/de/uka/ilkd/key/proof/rules/intRulesUncheckedSemantics.key

```diff
@@ -50,6 +50,9 @@
         \replacewith(true)
         \heuristics(concrete)
     };
+}
+
+\rules(programRules:Java & (intRules:arithmeticSemanticsIgnoringOF | intRules:arithmeticSemanticsCheckingOF)) {
```

Comment:
This leads to the question of whether we might want to have rules predicate so that we can easily model "X" or "not X" in the Taclet base.

Reply:
I do not understand your comment. Is it a concrete suggestion for this PR? If not, I would like to postpone it, since we should not wait any longer with the overflow fix.